server/controllers/RelatedItemsController.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+
+const router = express.Router();
+const axios = require('axios');
+
+const API_URL = process.env.API_ENDPOINT;
+const API_KEY = process.env.TOKEN;
+
+// API request for related products based off a product ID
+router.get('/:id/related', (req, res) => {
```

Comment:
It looks like you're trying to access the api route. Maybe all you need is '/:id'.
```suggestion
router.get('/:id', (req, res) => {
```